'''

[실습9 해설]위기탈출 넘버원

위기탈출 넘버원
break문으로 당신의 소중한 코드를 무한루프로부터 지킬 수 있습니다!

평소에도 코딩을 열심히 하던 준배. 그는 오늘도 엘리스에서 문제를 풀기 시작합니다.

그런데! 코드를 다 짜고 보니 코드가 멈추지 않고 계속 출력됩니다.

분명 맞게 작성한 것 같은데, 계속 ans가 출력이 안되는 걸 본 준배는 결국 혈압 상승으로 인해 응급실에 실려가게 되었습니다.

과연 무엇이 문제였을까요? 오른쪽 준배의 코드를 보고 잘못된 부분을 찾아 고쳐줍시다.

원래 문제

while문을 이용해서 1부터 10까지의 합을 구해서 이를 출력해봅시다.


이렇게 해봐요!
9번째 줄의 미션을 확인하고, 이를 수행해보세요.
실행 버튼을 눌러서 주어진 임무를 잘 수행했는지 확인하세요.
제출 버튼을 눌러 맞게 코드를 작성했는지 확인해보세요!

'''
# 다음은 while True:를 이용하여 준배가 짠 코드입니다.
i = 1
ans = 0
while True:
    ans = ans+i
    i = i+1
    if i > 10:
        break
print(ans)
